get-started/fine-tuning/platform.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Fine-tuning with the Platform
+description: Learn how to create custom models tailored to your specific tasks by fine-tuning foundation models with your own data using the kluster.ai visual interface.
+---
+
+# Fine-tuning with the Platform
+
+The kluster.ai platform provides a visual, no-code approach to fine-tuning AI models. With an intuitive interface and real-time feedback, you can train customized models without writing a single line of code.
+
+This guide walks you through the platform's fine-tuning workflow, from uploading your training data to deploying your specialized model.
+
+## Prerequisites
+
+Before getting started with fine-tuning, ensure you have the following:
+
+- **kluster.ai account**: Sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=_blank} if you do not have one.
```

Comment:
Use the snippet but with the line capture feature?

Reply:
Done!